etc/db_schema.xml

```diff
@@ -14,11 +14,12 @@
             <column name="product_id"/>
         </index>
     </table>
-    <table name="webhook_queue" resource="default" engine="innodb" comment="Webhook Queue">
+    <table name="balance_queue" resource="default" engine="innodb" comment="Balance Queue">
         <column xsi:type="int" name="entity_id" unsigned="false" nullable="false" identity="true" comment="Entity Id"/>
         <column xsi:type="int" name="attempts" nullable="true" comment="Attempts"/>
         <column xsi:type="text" name="payload" nullable="true" comment="Payload"/>
         <column xsi:type="varchar" name="name" nullable="true" comment="Webhook Name"/>
+        <column xsi:type="smallint" name="status" nullable="true" default="0" comment="0 => pending (default), 1 => in_progress, 2 => completed, 3 => failed"/>
```

Comment:
can we change this to enum? instead of 1,2,3 having `pending`, `in_progress`, `completed` and `failed`

Reply:
magento 2 doesnt recomment enum type but we can use the varchar here and add the string as values.

Reply:
let's go with varchar – it will make db more readable